GVFS/GVFS/Program.cs

```diff
@@ -89,6 +90,12 @@ public static void Main(string[] args)
                             upgrade.Execute();
                             Environment.Exit((int)ReturnCode.Success);
                         })
+                    .WithParsed<ConfigVerb>(

```

Comment:
`ServiceVerb`, `UpgradeVerb`, and now `ConfigVerb` all follow the same pattern - they are tied to machine state, not a particular repo. Please make a subclass like `GVFSVerb.NonRepoVerb` (see `GVFSVerb.ForExistingEnlistment` as an example) so that we can just have one `.WithParsed<GVFSVerb.NonRepoVerb>` here, and also eliminate the boilerplate in the implementations of all of those verbs.

Reply:
For a consistent name, perhaps go for `GVFSVerb.ForNoEnlistment`. Or something else if you have a better suggestion.

Reply:
Renamed to GVFSVerb.ForNoEnlistment